PyTDF/CallableGenerator.py

```diff
@@ -53,21 +54,21 @@ def _dfs(self, node, prev = 't', ops = []):
                 self._dfs(n, *new_tuple)
 
 
-    def get_mapper_callable(self, root_node):
+    def get_mapper_callable(self):
```

Comment:
I suggest we just call this `get_callable(self):`, the concept of mapper is only related to the way some backends will handle this. This is more generic, it is intended to get us a callable to which we can pass an RDF and it will apply all the operations of the graph to it.

Reply:
+1

Reply:
@shravan97 ping

Reply:
Oh yes. I'm sorry. I'll push a commit immediately